tests/foreman/api/test_errata.py

```diff
@@ -239,6 +241,45 @@ def test_positive_install_in_host(self):
             self._validate_package_installed([client], FAKE_2_CUSTOM_PACKAGE)
 
     @tier3
+    def test_positive_install_multiple_in_host(self):
+        """For a host with multiple applicable errata install one and ensure
+        the rest of errata is still available
+
+        :id: 67b7e95b-9809-455a-a74e-f1815cc537fc
+
+        :BZ: 1469800
+
+        :expectedresults: errata installation task succeeded, available errata
+            counter decreased by one; it's possible to schedule another errata
+            installation
+
+        :CaseLevel: System
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as client:
+            client.install_katello_ca()
+            client.register_contenthost(
+                self.org.label, self.activation_key.name)
+            self.assertTrue(client.subscribed)
+            client.enable_repo(REPOS['rhst7']['id'])
+            client.install_katello_agent()
+            host = entities.Host().search(query={
+                'search': 'name={0}'.format(client.hostname)})[0]
+            for package in FAKE_9_YUM_OUTDATED_PACKAGES:
+                self._install_package([client], [host.id], package)
+            host = host.read()
+            applicable_errata_count = host.content_facet_attributes[
+                'errata_counts']['total']
+            self.assertGreater(applicable_errata_count, 1)
```

Comment:
perhaps applicable_errata_count should be greater than 2 to be sure that "it's possible to schedule another errata  installation"
and the code became
```python
errata_to_install = 2
self.assertGreater(applicable_errata_count, errata_to_install)
for errata in FAKE_9_YUM_ERRATUM[:errata_to_install]:
    ...
```
what do you think?

Reply:
`> 1` means at least 2. 2 is enough to schedule second errata installation

Reply:
I was thinking about the second last one that it has errata to apply, (for the first one we apply the second, but for the second any way remain some errata but we do not apply them), that what I understood from that expectedresults, 
but with the description we can also understand what you have implemented so ACK